cypress-cucumber/cypress/e2e/API/Books/books.cy.js

```diff
@@ -36,17 +36,10 @@ class Books {
         return cy.request('GET', baseUrl + '/api/books');
     }
 
-    getBook(id, authHeader) {
```

Comment:
Duplicate declaration.

Reply:
@shiran-dinushka is this ok to remove

Reply:
It is ok. I forgot to remove authHeader parameter